docs/install.md

```diff
@@ -26,7 +26,7 @@ This guide explains how to install Tekton Pipelines. It covers the following top
    ```bash
    #Example command for creating a cluster on GKE
    gcloud container clusters create $CLUSTER_NAME \
-     --zone=$CLUSTER_ZONE
+     --zone=$CLUSTER_ZONE --cluster-version=latest
```

Comment:
Thanks for this.

It does say version 1.15 or later on L24, but it is better to have a working command here.
NIT: `latest` here will change over time, depending on what GKE runs, so perhaps we should use `--cluster-version`. On the other side a specific version may not be available anymore at some point in time, but still I slightly  prefer a specific version.

Reply:
yeah, I think a specific version would be nice!

Reply:
ok can someone suggest a version?

Reply:
I would say the latest available now `1.15.11-gke.5`